pkg/controller/machine/bootstrap.go

```diff
@@ -87,17 +89,24 @@ func getOSMBootstrapUserDataForIgnition(req plugin.UserDataRequest, sshPublicKey
 		return "", fmt.Errorf("failed to parse bootstrap-ignition-config template: %v", err)
 	}
 
+	bootstrapKubeconfig, err := helper.StringifyKubeconfig(req.Kubeconfig)
+	if err != nil {
+		return "", fmt.Errorf("failed to format bootstrap kubeconfig: %v", err)
+	}
```

Comment:
Can you please move this to `getOSMBootstrapUserdata` and then just pass the value down to `getOSMBootstrapUserDataForIgnition` and `getOSMBootstrapUserDataForCloudInit`? 

Just a nit to avoid duplication